features/react-component-check.feature

```diff
@@ -0,0 +1,86 @@
+Feature: React Component Check
+  In order to have confidence in my site's React components
+  As an anonymous user
+  I want to verify that each component type successfully initializes.
+
+  @api @javascript
+  Scenario: Verify Card Arrangements (components-core)
+    Given I am an anonymous user
+    When I am at '/card-arrangement'
+    Then I should see the heading "Card Arrangement"
+    Then I should see the heading "Two Columns"
+    Then I should see the heading "Three Columns"
+    Then I should see the heading "Four columns"
+    Then I should see the heading "Vertical Arrangement"
+    Then I should see the heading "Ranking Cards"
+    Then I should see the heading "Image based cards"
```

Comment:
@juanmitriatti Verifying using the headings is probably not a good test because Drupal is handling the rendering of the headings. We should be checking for something inside  the container the component renders in.

Reply:
Perhaps a check that the container isn't empty would be an easily repeatable check.

Reply:
This is accomplished on line 17, which tests that `multiple "[data-testid='card-container']" elements exist`. These would not exist if the component didn't load. But, I can make it more specific.